src/main/java/mekanism/common/tile/transmitter/TileEntitySidedPipe.java

```diff
@@ -479,6 +488,10 @@ public void onAdded() {
     @Override
     public void onLoad() {
         onWorldJoin();
+        if (getPossibleTransmitterConnections() != currentTransmitterConnections) {
+            //Mark the transmitters as invalidated if they do not match what we have stored/calculated
+            markDirtyTransmitters();
```

Comment:
I'm not sure `markDirtyTransmitters` is the right method to call here, as it mostly notifies neighbours of a change. Perhaps `refreshConnections()` would be more appropriate?

Seems otherwise it's relying on an adjacent transmitter to call the updateNeighbour path

Reply:
Your right `refreshConnections()` does seem to behave a lot better here.